test/jasmine/tests/plot_api_test.js

```diff
@@ -348,7 +384,45 @@ describe('Test plot api', function() {
                 expect(gd.data[0].zmax).toBe(10);
 
                 expect(gd.data[0].contours).toEqual({start: 3, end: 9, size: 1});
-            });
+            })
+            .catch(fail)
+            .then(done);
```

Comment:
Hah, this test wasn't actually running. More precisely, it was running but not being used. Who wrote that sloppy thing? 🙈